dotcom-rendering/src/web/components/Card/Card.stories.tsx

```diff
@@ -0,0 +1,382 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import { css } from '@emotion/react';
+import {
+	ArticleDesign,
+	ArticleDisplay,
+	ArticlePillar,
+	ArticleSpecial,
+} from '@guardian/libs';
+import { breakpoints, from } from '@guardian/source-foundations';
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { Card } from './Card';
+
+const basicCardProps = {
```

Comment:
If you exported the `Props` type from `Card.tsx`, you could use set this type to it, and avoid the `SmallHeadlineSize` and `ImagePositionType` assertions

Reply:
Sensible. Done